maestrowf/maestro.py

```diff
@@ -67,25 +67,35 @@ def status_study(args):
     directory_list = args.directory
 
     if directory_list:
-        header_format = "".ljust(150, "=")
 
         for path in directory_list:
             abs_path = os.path.abspath(path)
 
-            print(header_format)
-            print(abs_path)
-            print(header_format)
-
             status = Conductor.get_status(abs_path)
+            status_layout = args.layout
+
             if status:
-                print(tabulate.tabulate(status, headers="keys"))
+                try:
+                    # Wasteful to not reuse this renderer for all paths?
```

Comment:
In terms of wasteful, probably; but, for the scales that we're at I don't think this is a limiting factor.